### ISSUE TITLE
Handling errors

### ISSUE THREAD

**Issue by vaeng** [exercism/sqlite#20]
I have an idea for handling errors.

Normally you would deny faulty data to be inserted into the database before it happens.
Currently, we have a start table, that has valid data. We could change this method to insert one row after another and let the students create checks and catch the error message somehow.
I am unsure how obvious these error test cases would be for those who read our test.sql file.

As an alternative, we could ask them to clean erroneous data from the table and check if it is still there.

Do you have other ideas?

**Comment by IsaacG** [comment]
We could ask them to mark data as valid or not, but that doesn't feel like the spirit of the exercise. 

We could also pass if it's not a great fit.

**Comment by vaeng** [comment]
I have an example of error handling implemented in the nucleotide count exercise. Maybe we can use that style for other exercises as well

https://github.com/exercism/sqlite/blob/main/exercises/practice/nucleotide-count/nucleotide-count_test.sql

**Comment by IsaacG** [comment]
Yeah. TRIGGER and RAISE would definitely be a good concept to include in a syllabus if/when we build one out. I'm not sure they are necessary to roll out the exercises with errors, as you've already demonstrated.
